sites/all/modules/reol_field_carousel/reol_field_carousel.module

```diff
@@ -195,11 +201,44 @@ function reol_field_carousel_field_widget_form(&$form, &$form_state, $field, $in
     '#default_value' => isset($items[$delta]['search']) ? $items[$delta]['search'] : NULL,
     '#cols' => 60,
     '#rows' => 5,
+    '#states' => array(
+       'required' => array(
+         ':input[name="field_inspiration_paragraphs[und][0][field_carousel][und]['.$delta.'][title]"]' => array('!value' => ''),
+       ),
+    ),
+    '#element_validate' => array('reol_field_carousel_field_widget_form_validate'),
   );
 
   return $element;
 }
 
+/**
```

Comment:
Would is not be possible to just add `"#required" => true` to the form elements?

Reply:
@cableman, see https://github.com/eReolen/base/pull/277/commits/de3074dd47f948d07e1f95e975a6b8bac501a260.